alcs-frontend/src/app/features/search/file-type-filter-drop-down/file-type-filter-drop-down.component.html

```diff
@@ -1,13 +1,21 @@
-<mat-form-field appearance="outline"  class="file-type-input">
+<mat-form-field
+  matTooltip="Note: This field searches both proposal and decision component type"
+  appearance="outline"
+  class="file-type-input"
+>
+  <mat-label>File Type</mat-label>
   <input
     value="{{ getSelectedItems() }}"
-    placeholder="File Type"
+    id="type"
     type="text"
     matInput
     [matAutocomplete]="auto"
     onkeydown="return false;"
     [formControl]="componentTypeControl"
   />
+  <button matSuffix *ngIf="hasItemsSelected" class="clear-button" type="button" (click)="clear()">×
```

Comment:
In a previous PR, you used `<mat-icon>close</mat-icon>` for an 'x' button. Should we do the same here for consistency?

Reply:
Good suggestion! Not sure why I didn't. Updated